diff --git a/GridWorld.py b/GridWorld.py
index 8ecf3e0..593dfcc 100644
--- a/GridWorld.py
+++ b/GridWorld.py
@@ -148,7 +148,7 @@ class GridEnv(gym.Env):
         ls = LightSource(270, 45)
         rgb = ls.shade(self.z, cmap=cm.gist_earth, vert_exag=0.1, blend_mode='soft')
         surf = ax.plot_surface(Y, X, self.z, rstride=1, cstride=1, facecolors=rgb,
-                            linewidth=0, antialiased=False, alpha=0.6,shade=False)
+                            linewidth=0, antialiased=False, alpha=0.9,shade=False)
         ax.set_xlabel('X Label')
         ax.set_ylabel('Y Label')
         ax.set_zlabel('Z Label')
@@ -166,5 +166,5 @@ class GridEnv(gym.Env):
         plt.pause(0.001)  
         plt.close()  
 
-# grid = GridEnv()
+grid = GridEnv()
 # grid.my_render()
\ No newline at end of file
diff --git a/PPOagent.py b/PPOagent.py
index ee6b62b..1934183 100644
--- a/PPOagent.py
+++ b/PPOagent.py
@@ -18,7 +18,7 @@ def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--exp-name", type=str, default=os.path.basename(__file__).rstrip(".py"),
         help="the name of this experiment")
-    parser.add_argument("--seed", type=int, default=3406,
+    parser.add_argument("--seed", type=int, default=2022,
         help="seed of the experiment")
     parser.add_argument("--torch-deterministic", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
         help="if toggled, `torch.backends.cudnn.deterministic=False`")
@@ -36,7 +36,7 @@ def parse_args():
     # Algorithm specific arguments
     parser.add_argument("--env-id", type=str, default="GridWorld-v0",
         help="the id of the environment")
-    parser.add_argument("--total-timesteps", type=int, default=10000000,
+    parser.add_argument("--total-timesteps", type=int, default=9000000,
         help="total timesteps of the experiments")
     parser.add_argument("--learning-rate", type=float, default=2.5e-4,
         help="the learning rate of the optimizer")
diff --git a/PPOinference.py b/PPOinference.py
index 6e5a684..c743640 100644
--- a/PPOinference.py
+++ b/PPOinference.py
@@ -158,8 +158,8 @@ if __name__ == "__main__":
             # env.my_render()
             sum_reward += reward
             if done:
-                print(f"episode={ep}, episodic_return={info['episode']['r']}")    
+                print(f"episode={ep}, episodic_return={info['episode']['r']}," f"episodic_length={info['episode']['l']}")    
                 print('Sum reward = ',sum_reward)
                 break
-
+        print('crashed',env.crashed)
     env.close()
\ No newline at end of file
diff --git a/models/actor_seed_3406.pth b/models/actor_seed_3406.pth
deleted file mode 100644
index 1f6e41c..0000000
Binary files a/models/actor_seed_3406.pth and /dev/null differ
diff --git a/models/actor_seed_3407.pth b/models/actor_seed_3407.pth
deleted file mode 100644
index fc03e21..0000000
Binary files a/models/actor_seed_3407.pth and /dev/null differ
diff --git a/models/critic_seed_3406.pth b/models/critic_seed_3406.pth
deleted file mode 100644
index 97a3653..0000000
Binary files a/models/critic_seed_3406.pth and /dev/null differ
diff --git a/models/critic_seed_3407.pth b/models/critic_seed_3407.pth
deleted file mode 100644
index 0bcea43..0000000
Binary files a/models/critic_seed_3407.pth and /dev/null differ
